### DISCUSSION TITLE
Do we abandon the plugins ```from pytorch_lightning.plugins import FullyShardedPlugin```

### DISCUSSION THREAD

**Discussion by ljz756245026** [Lightning-AI/pytorch-lightning#9619] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Recently, I am trying to run some DeepSpeed and FairScale example on Pytorch Lightning.And I found this blog (https://devblog.pytorchlightning.ai/accessible-multi-billion-parameter-model-training-with-pytorch-lightning-deepspeed-c9333ac3bb59) and these examples(https://github.com/SeanNaren/minGPT/blob/fairscale/benchmark.py). However, I cannot find the plugins:
```
from pytorch_lightning.plugins import FullyShardedPlugin
```
I want to know whether this plugin is on beta and not available.Where I can get these plugins and run these example?
Thank you!

**Comment by ananthsub** [comment]
you should be able to import `from pytorch_lightning.plugins import DDPFullyShardedPlugin`

https://github.com/PyTorchLightning/pytorch-lightning/blob/cc77367ffcf604074d16f3b57a4dcc4b0ca14702/pytorch_lightning/plugins/__init__.py#L47
